packages/eslint-config-kodefox/index.js

```diff
@@ -19,8 +19,8 @@ module.exports = {
   rules: {
     // TypeScript (@typescript-eslint/eslint-plugin)
     '@typescript-eslint/array-type': ['error', { default: 'generic' }],
-    '@typescript-eslint/camelcase': ['error', { properties: 'always' }],
```

Comment:
@sstur Is this in favor of [naming-convention](https://github.com/typescript-eslint/typescript-eslint/blob/5de66142f2391b000aa939b86d4756ee621553ad/packages/eslint-plugin/docs/rules/naming-convention.md) here?

Reply:
I think so, yes. I got a message saying the camelcase rule was deprecated.